cmd/helpers/mssql/mssql.go

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+package mssql
+
+import (
+	"fmt"
+	"github.com/microsoft/go-sqlcmd/cmd/sqlconfig"
+	"github.com/microsoft/go-sqlcmd/pkg/sqlcmd"
+	"os"
+)
+
+func Connect(endpoint sqlconfig.Endpoint, user sqlconfig.User, console sqlcmd.Console) *sqlcmd.Sqlcmd {
+	// BUG(stuartpa): Check what the value of fromEnvironment should be here
```

Comment:
>(stuartpa): Check what the valu [](http://example.com/codeflow?start=7&length=31)

Using false or true depends on whether you want sqlcmd to pick up variables for formatting the output.

Reply:
Ok, I've set to True